webpack.config.js

```diff
@@ -22,20 +24,27 @@ module.exports = {
             {test: /\.js$/, exclude: /node_modules/, loader: 'babel-loader'}
         ]
     },
+    optimization: {
+        minimizer: [
+            new UglifyJSPlugin({
+                parallel: require('os').cpus().length,
+                uglifyOptions: {
+                    ie8: false,
+                    keep_fnames: false,
+                    output: {
+                        beautify: false,
+                        comments: false
+                    }
+                }
+            })
+        ]
+    },
     plugins: production ? [
-            new webpack.optimize.UglifyJsPlugin({
-                beautify: false,
-                mangle: {
-                    screw_ie8: true,
-                    keep_fnames: true
-                },
-                compress: {
-                    screw_ie8: true
-                },
-                comments: false
-            }),
-            new webpack.BannerPlugin({banner})
+            new webpack.BannerPlugin({ banner }),
+            new BundleAnalyzerPlugin({
+                analyzerMode: process.env.ANALYZE_BUIlD ? 'server' : 'disabled'
```

Comment:
Shouldn't it be `ANALYZE_BUILD` instead ?

Reply:
Can you explain?

Reply:
`l` is not capitalised 🤔 so it would be `ANALYZE_BUILD` instead of `ANALYZE_BUIlD`.

Makes it easier to read imo

Reply:
Oh, thank you. Let me fix that